cocos/asset/asset-manager/parser.ts

```diff
@@ -173,124 +81,15 @@ export class Parser {
     }
 
     public parsePVRTex (file: ArrayBuffer | ArrayBufferView, options: IDownloadParseOptions, onComplete: CompleteCallback<IMemoryImageSource>) {
-        let err: Error | null = null;
-        let out: IMemoryImageSource | null = null;
-        try {
-            const buffer = file instanceof ArrayBuffer ? file : file.buffer;
-            // Get a view of the arrayBuffer that represents the DDS header.
-            const header = new Int32Array(buffer, 0, PVR_HEADER_LENGTH);
-
-            // Do some sanity checks to make sure this is a valid DDS file.
-            if (header[PVR_HEADER_MAGIC] === PVR_MAGIC) {
-                // Gather other basic metrics and a view of the raw the DXT data.
-                const width = header[PVR_HEADER_WIDTH];
-                const height = header[PVR_HEADER_HEIGHT];
-                const dataOffset = header[PVR_HEADER_METADATA] + 52;
-                // todo: use new Uint8Array(buffer, dataOffset) instead
-                // buffer = buffer.slice(dataOffset, buffer.byteLength);
-                const pvrtcData = new Uint8Array(buffer, dataOffset);
-                out = {
-                    _data: pvrtcData,
-                    _compressed: true,
-                    width,
-                    height,
-                    format: 0,
-                };
-            } else if (header[11] === 0x21525650) {
-                const headerLength = header[0];
-                const height = header[1];
-                const width = header[2];
-                // todo: use new Uint8Array(buffer, headerLength) instead
-                // buffer = buffer.slice(headerLength, buffer.byteLength);
-                const pvrtcData = new Uint8Array(buffer, headerLength);
-                out = {
-                    _data: pvrtcData,
-                    _compressed: true,
-                    width,
-                    height,
-                    format: 0,
-                };
-            } else {
-                throw new Error('Invalid magic number in PVR header');
-            }
-        } catch (e) {
-            err = e as Error;
-        }
-        onComplete(err, out);
+        ImageAsset.parseCompressedTexs(file, 0);
```

Comment:
Not need to invoke onComplete callback now?

Reply:
@holycanvas Do we need to use onComplete callback to here ?